troxileia.yaml

```diff
@@ -0,0 +1,18 @@
+FROM centos:centos7
+MAINTAINER Stech Training
+
+RUN yum -y update \
+    #Install Python repo, Python 3.6  and some addons
+    && yum -y install https://centos7.iuscommunity.org/ius-release.rpm \
+    && yum -y install python36u \
+    && yum -y install python36u-libs \
+    && yum -y install python36u-devel \
+    && yum -y install python36u-pip \
+    && yum -y clean all \
+    #Install Python Kubernetes client lib.
+    && python3.6 -m pip install kubernetes paramiko
+ENV PYTHONUNBUFFERED=0
+#Put script inside container
+COPY sensor_controller.py pod_controller.py gigamon.py main.py ./
+#When the container starts run the python controller
+CMD ["python3.6", "main.py"]

```

Comment:
seems like a duplicate of the Dockerfile, not sure if you need this or if it's just the wrong file contents